src/Microsoft.ML.Featurizers/DateTimeTransformer.cs

```diff
@@ -675,7 +687,18 @@ internal override unsafe void CreateTransformerFromEstimator(DateTimeEstimator.H
             private static unsafe extern bool CreateTransformerFromSavedDataNative(byte* rawData, IntPtr bufferSize, byte* dataRootDir, out IntPtr transformer, out IntPtr errorHandle);
             private protected override unsafe void CreateTransformerFromSavedDataHelper(byte* rawData, IntPtr dataSize)
             {
-                fixed (byte* dataRootDir = Encoding.UTF8.GetBytes(AppDomain.CurrentDomain.BaseDirectory + char.MinValue))
+                byte[] dataRoot;
+
+                if (Directory.Exists(AppDomain.CurrentDomain.BaseDirectory + "/Data/DateTimeFeaturizer"))
+                {
+                    dataRoot = Encoding.UTF8.GetBytes(AppDomain.CurrentDomain.BaseDirectory + char.MinValue);
+                }
+                else
+                {
+                    dataRoot = Encoding.UTF8.GetBytes(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) + char.MinValue);
+                }
+
+                fixed (byte* dataRootDir = dataRoot)
                 {
```

Comment:
You mentioned this was fixed before? Is this only a temporary solution? If so, could add a comment saying so?

Reply:
This isn't a temporary fix, see the prior comment I just responded to.